repose-aggregator/src/docs/asciidoc/_templates/filter.adoc

```diff
@@ -0,0 +1,57 @@
+= Some Filter
+
+include::../_includes/in-progress.adoc[]
+
+This filter does something really exciting.
+
+
+== General filter information
+* *Name:* some
+* *Default Configuration:* some.cfg.xml
+* *Released:* v9001.0.0.0
+* *Bundle:* repose-filter-bundle
+* link:../schemas/some.xsd[Schema]
```

Comment:
Lol, I was surprised it didn't exist yet, so I made it. So it's settledish for now.